opal/mca/base/help-mca-base.txt

```diff
@@ -59,3 +59,9 @@ all components *except* a and b", while "c,d" specifies the inclusive
 behavior and means "use *only* components c and d."
 
 You cannot mix inclusive and exclusive behavior.
+[framework-param:exclude-always-component]
+A request was made to exclude a component from consideration that
+must always be considered.
+
```

Comment:
Let's make this super clear for the user (per George's comment) that this component will be considered anyway.

Maybe change to:
```text
Warning: A request was made to exclude a component from consideration that
must always be considered. This component (noted below) will -not- be excluded
from consideration. The program will continue running with this component being
made available for selection.

Framework: %s
Component: %s
```

What do you think?